### PR TITLE
stripe-v3: Add disableLink

### PR DESCRIPTION
**Pull request by qqilihq**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/payments/link-card-element#enable-link
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/stripe-v3/index.d.ts [hunk 1]
```diff
@@ -1024,6 +1025,7 @@ declare namespace stripe {
             paymentRequest?: paymentRequest.StripePaymentRequest | undefined;
             supportedCountries?: string[] | undefined;
             disabled?: boolean | undefined;
+            disableLink?: boolean | undefined;
```
Comment (kaznovac) [inline types/stripe-v3/index.d.ts:1028]: IMHO: there is no need to add the explicit `undefined` type, but it respects the consistency of the code.

File: types/stripe-v3/stripe-v3-tests.ts [hunk 2]
```diff
@@ -42,6 +42,7 @@ describe("Stripe elements", () => {
             value: {
                 postalCode: '94110',
             },
+            disableLink: true
```
Comment (kaznovac) [inline types/stripe-v3/stripe-v3-tests.ts:45]:
```suggestion
            disableLink: true,
```
Add the trailing comma here please.

Trailing commas reduce the diff for the next PR's code review

**Review by kaznovac — APPROVED** [review]
@qqilihq welcome :)

Just check the box regarding the tests in PR's description

And for you information please note that `stripe-js` has the official type definitions https://github.com/stripe/stripe-js

**Comment by qqilihq** [comment]
@kaznovac 

> And for you information please note that stripe-js has the official type definitions https://github.com/stripe/stripe-js

Thank you, I wasn't aware of that! 👍

**Review by kaznovac — APPROVED** [review]

**Comment by qqilihq** [comment]
Ready to merge

**Pull request merged**